lib/publisher.ex

```diff
@@ -231,7 +230,7 @@ defmodule GenRMQ.Publisher do
   end
 
   defp app_id(config) do
-    config[:app_id] || Keyword.get(Project.config(), :app)
+    config[:app_id]
```

Comment:
Let's use `:gen_rmq` as the default if there's no config[:app_id]

Reply:
I've updated the Pull Request